ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/Send.hs

```diff
@@ -17,18 +21,21 @@ import Graphics.UI.Qtah.Core.HSize (HSize(..))
 import qualified Graphics.UI.Qtah.Core.QEvent as QEvent
 import qualified Graphics.UI.Qtah.Core.QLocale as QLocale
 import qualified Graphics.UI.Qtah.Core.QObject as QObject
-import Graphics.UI.Qtah.Core.Types (QtWindowType(Popup))
+import Graphics.UI.Qtah.Core.Types (QtWindowType(Popup), alignRight)
```

Comment:
I prefer to place unqualified imports before qualified once. Glue this to `connect_` import above.